suite/tests/client-interface/translate_sandbox.c

```diff
@@ -0,0 +1,153 @@
+/* **********************************************************
+ * Copyright (c) 2017 Simorfo, Inc.  All rights reserved.
```

Comment:
nit: since several pieces were copied from existing code there should also be a google copyright line